src/Sylius/Bundle/ApiBundle/Command/Admin/SendResetPasswordEmail.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ *  This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Command\Admin;
+
+use Sylius\Bundle\ApiBundle\Command\IriToIdentifierConversionAwareInterface;
+use Sylius\Bundle\ApiBundle\Command\LocaleCodeAwareInterface;
+
+/** @experimental */
+class SendResetPasswordEmail implements LocaleCodeAwareInterface, IriToIdentifierConversionAwareInterface
```

Comment:
Why do you need these interfaces? This command will never be used from API 
```suggestion
class SendResetPasswordEmail
```